app/assets/javascripts/discourse/app/controllers/login.js

```diff
@@ -297,56 +352,6 @@ export default Controller.extend(ModalFunctionality, {
       this.send("showCreateAccount");
     },
 
-    forgotPassword() {
```

Comment:
One plugin is using `this.send("forgotPassword")` - can you confirm `send()` still works with `@action`?

cc: @davidtaylorhq (it's f-profile-sync)

Reply:
I've confirmed that `this.send("forgotPassword")` will invoke `@action forgotPassword` within either a component or a controller.